Corona/Data/DataManager.swift

```diff
@@ -103,6 +110,11 @@ extension DataManager {
 			countries.first { $0.name == "US" }?.timeSeries = timeSeriesRegion.timeSeries
 		}
 
+		/// Update Germany time series
```

Comment:
Does the new data source have a time-series data for Germany? If not, you don't have to do this.

Reply:
Remove this and see how it amazingly crashes when you force unwrap timeSeries ;-)

Reply:
Oh okay. Then I have to fix that issue first. I'll look into it after merging.